cf-console-header-menu-button-sample/src/aem-cf-console-admin-1/web-src/src/components/UIExtensibilityModal.js

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React, { useState, useEffect, useCallback } from "react";
+import { attach } from "@adobe/uix-guest";
+import {
+  Heading,
+  Provider,
+  Content,
+  Divider,
+  defaultTheme,
+  ButtonGroup,
+  Button,
+  Flex,
+  View,
+  Text
+} from "@adobe/react-spectrum";
+import Info from "@spectrum-icons/workflow/Info";
+import "./UIExtensibilityModal.css";
+
+import { extensionId } from "./Constants";
+
+export default UIExtensibilityModal = () => {
+  const [guestConnection, setGuestConnection] = useState(null);
+  const [sharedContext, setSharedContext] = useState(null);
+
+  useEffect(() => {
+    const init = async () => {
+      const connection = await attach({ id: extensionId });
+      setGuestConnection(connection);
+      setSharedContext(connection.sharedContext);
+    };
+
+    init().catch(console.error);
+  }, []);
+
+  const displayToast = useCallback((toast, variant) => {
+    if (guestConnection !== null) {
+      guestConnection.host.toaster.display({
+        variant,
+        message: toast,
+        timeout: 1500,
+      });
+    }
+  }, [guestConnection]);
+
+  const onCloseHandler = () => {
+    guestConnection.host.modal.close();
+  };
+
+  return (
+    <Provider theme={defaultTheme} colorScheme="light">
+      <Content width="100%">
+        <Heading level={3}>Check out some of AEM UI Extensibility features</Heading>
+        <View marginBottom="size-150">
```

Comment:
Need to add some explanation here, like "You can use Adobe React Spectrum Toast component to trigger in-app notifications."